runtime/configs/config.go

```diff
@@ -28,6 +29,10 @@ type Config struct {
 	Logger      *LoggerConfig      `json:"logger"`
 	RPC         *RPCConfig         `json:"rpc"`
 	Keybase     *KeybaseConfig     `json:"keybase"` // Determines and configures which keybase to use, `file` or `vault`. IMPROVE(#626): See for rationale around proto design. We have proposed a better config design, but did not implement it due to viper limitations
+	Validator   *ValidatorConfig   `json:"validator"`
+	Servicer    *ServicerConfig    `json:"servicer"`
+	Fisherman   *FishermanConfig   `json:"fisherman"`
+	Portal      *PortalConfig      `json:"portal"`
```

Comment:
I think we should either remove `Portal` from this PR completely or make sure it's account for everything.

For example:
- `runtime/genesis/proto/genesis.proto`
- `ActorType` in `actor.proto`

Personally, I think one PR to add Portal everywhere later is the way to go